.storybook/config.js

```diff
@@ -8,5 +9,13 @@ setOptions({
     url: 'https://github.com/Doist/reactist',
 })
 
-const loadStories = () => require('../stories')
+addDecorator(withKnobs)
+
+const req = require.context('../src', true, /.stories.tsx$/)
+
+function loadStories() {
+    require('../stories')
+    req.keys().forEach((file) => req(file))
```

Comment:
If we want to keep using the `new-components` folder, it'd be nice to sort the component path here so the components are still sorted alphabetically in storybook.

Reply:
I sorta get what you mean, but not quite exactly. Do you mind contributing that change? Seems small and you have the context because it is your idea. (unless what you mean is a simple `.sort()` on this array before calling `forEach()`).